### PR TITLE
Adding section for updating the CSS with API REST

### PR DESCRIPTION
**Pull request by sedkis**

### CODE REVIEW

**Comment by marksou** [comment]
@sedkis do we need to explain that users need to change the URL in the curl commands to their own portal URL?

**Comment by nathanschepers** [comment]
I talked to @lonelycode and he said we can also do page content customization via API calls. I haven't been able to find these documented anywhere. Should that be part of this?

**Comment by sedkis** [comment]
@nathanschepers not a part of it, but we definitely need to add something similar

File: tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md [hunk 1]
```diff
@@ -26,6 +26,39 @@ In the CSS Editor, add the classes that you would like to override in the home p
 
 If you wish to customise how emails are displayed to end-users, then you can also add new classes to the Email CSS editor, these classes will be added in-line to the email that is sent out:
 
+### Updating CSS With API
```
Comment (asoorm) [inline tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md]: change `With` to `by` or `via`

File: tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md [hunk 2]
```diff
@@ -26,6 +26,39 @@ In the CSS Editor, add the classes that you would like to override in the home p
...
+Alternatively, you can update the CSS with an API call.  the below `curl` command will update the CSS for your organization.
```
Comment (asoorm) [inline tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md]: `the below ``curl`` command will` -> `The curl command below will`

Capital `T`.

**Comment by asoorm** [comment]
I would recommend replacing Auth `8f297c42596541034c37ea8dc9349fef` and org_id `5cca0452817e000001c2a543` with an env-var. That way end-users can copy and paste your code snippets.

File: tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md [hunk 3]
```diff
@@ -26,6 +26,39 @@ In the CSS Editor, add the classes that you would like to override in the home p
...
+    "org_id": "5cca0452817e000001c2a543",
+    "page_css": ".btn-success {background-color: magenta}"
+  }' | python -mjson.tool
+```
+
+To get the `CSS_BLOCK_ID`, run this `curl` command:
+
+```{.copyWrapper}
+curl www.tyk-test.com:3000/api/portal/css \
+-H "Authorization:8f297c42596541034c37ea8dc9349fef" | python -mjson.tool
```
Comment (asoorm) [inline tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md]: I would remove the `| python -mjson.tool`. it makes output pretty, but is not relevant for the documentation.

**Comment by sedkis** [comment]
thanks @asoorm for quick turnaround

File: tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md [hunk 4]
```diff
@@ -26,6 +26,39 @@ In the CSS Editor, add the classes that you would like to override in the home p
...
+  -H "Content-Type: application/json" \
+  -d '{
+    "email_css": "",
+    "id":{CSS_BLOCK_ID},
+    "org_id": "5cca0452817e000001c2a543",
+    "page_css": ".btn-success {background-color: magenta}"
+  }' | python -mjson.tool
+```
+
+To get the `CSS_BLOCK_ID`, run this `curl` command:
```
Comment (letzya) [inline tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md]: I would recommend rewriting the order. You can explain that as in any update of objects in a data store, you need to supply the object_id of the CSS block in the update api call so the flow is:
1. Obtain the CCS block id....
2. Update the CSS block...

Comment (sedkis) [reply tyk-docs/content/tyk-developer-portal/customise/customising-using-dashboard.md]: Yes this makes sense, I'll do this

**Comment by sedkis** [comment]
@asoorm I've updated the pages to use what @marksou usually does, based off of:
https://github.com/TykTechnologies/tyk-docs/pull/847#discussion_r304512695

So `{DASHBOARD_API_KEY}` 

If this is good, please approve and merge

**Review by letzya — APPROVED** [review]

**Pull request merged**